calculator.rb

```diff
@@ -0,0 +1,75 @@
+class CalculatorNumbers
+    def get_values
+        gets.split(" ").map{ |s| s.chomp.to_i }
+    end
+       
+    def operators operator, list_of_num
+        response = list_of_num.inject(operator)
+        puts "Result is: #{response}" 
+    end
+end
+    
+class Operate < CalculatorNumbers
```

Comment:
A class should be a noun, as you are making an object with them. So this would be Operation, rather than Operate.